lib/rouge/lexers/msgtrans.rb

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class MsgTrans < RegexLexer
+      title "MessageTrans"
+      desc "RISC OS message translator messages file"
+      tag 'msgtrans'
+      filenames 'Messages*'
+
+      state :root do
+        rule %r/^#[^\n]*/, Comment
+        rule %r/[^\n ,):?\/]+/, Name::Variable
```

Comment:
I'd usually expect a rule for a variable name to be based on `\w`. What's written here would match things like `\t`. Is that intentional?

Reply:
There's actually a Backus–Naur specification for this file format, and it explicitly allows any printing character other than those I excluded in the regex. Good point about `\t`, I checked the source code and it treats it as a control character, not a printing character. Same goes for `\r` I guess. How many control characters might we expect to encounter, and is there even a way to specify any other than `\n`, `\r` and `\t`?

Reply:
Right but a control character shouldn't be lexed as a variable name should it? That sounds like something best to tokenise as Text.

Reply:
I don't know what I thought I was saying with the previous comment but it doesn't make sense given your reply. Sorry for the confusion. Your new code looks good to me 👍